src/Providers/Trustpilot/Request/AccessTokenRequest.php

```diff
@@ -0,0 +1,77 @@
+<?php
+
+namespace PodPoint\Reviews\Providers\Trustpilot\Request;
+
+use GuzzleHttp\Psr7\Request;
+use PodPoint\Reviews\AccessToken;
+use PodPoint\Reviews\Request\BaseRequest;
+
+/**
+ * Class AccessTokenRequest
+ * @package PodPoint\Reviews\Providers\Trustpilot\Request
+ */
+class AccessTokenRequest extends BaseRequest
+{
+    const API_KEY = 'apiKey';
+    const API_SECRET = 'apiSecret';
+    const USERNAME = 'username';
+    const PASSWORD = 'password';
+
+    public function requiredFields(): array
+    {
+        return [
+            self::API_KEY,
+            self::API_SECRET,
+            self::USERNAME,
+            self::PASSWORD,
+        ];
+    }
+
+    /**
+     * Builds the request.
+     *
+     * @return Request
+     */
+    public function getRequest(): Request
+    {
+        $key = base64_encode($this->getOption(self::API_KEY) . ':' . $this->getOption(self::API_SECRET));
```

Comment:
this could use templating syntax also I think
something like
```
$key = base64_encode("{$this->getOption(self::API_KEY)}:{$this->getOption(self::API_SECRET)}");
```